### ISSUE TITLE
[Modular CLI]:  modular: error: invalid manifest: was not an object

### ISSUE THREAD

**Issue by raman1383** [modularml/mojo#629]
### Issue description

the  third step goes well:
In the Ubuntu terminal, install the Modular CLI:
curl https://get.modular.com | \
  MODULAR_AUTH=mut_cad487f14da24b91ab45c316d9f9a798 \
  sh -

and modular command works too.
but when trying to download mojo(step 4): modular install mojo, fails. 
it says:
modular: error: invalid manifest: was not an object

### Steps to reproduce

- Include relevant code snippet that did not work as expected.
- If applicable, add screenshots to help explain the problem.
- Include anything else that might help us debug the issue.

### Version Info

```shell
- Provide Modular CLI version by pasting the output of `modular -v`: modular 0.1.4 (6b54d308)
- What OS did you install modular CLI on ?  windows 11 on WSL 22
```

**Comment by raman1383** [comment]
I ran  `modular clean` before `modular install mojo`.
then after running `modular install mojo`:
# Found release for https://packages.modular.com/mojo @ 0.2.1, installing to /home/raman/.modular/pkg/packages.modular.com_mojo

**Comment by raman1383** [comment]
then after 10 minutes and 300MB:
==========================
Failure Information:
        - http error: failed to reach URL https://cas.modular.com/0/VXDD2K6eZBKVweei-2KEf4STd5NlFsCbeFWSn_FWyTg with cURL error SSL connect error - Looking for VXDD2K6eZBKVweei-2KEf4STd5NlFsCbeFWSn_FWyTg
        - http error: failed to reach URL https://cas.modular.com/0/LDgEhgiiTkQ4w95nYEo0hv500_aD36bTjTq1Lajcb8I with cURL error Failure when receiving data from the peer - Looking for LDgEhgiiTkQ4w95nYEo0hv500_aD36bTjTq1Lajcb8I
==========================

**Comment by bzcheeseman** [comment]
It looks like your connection may have been interrupted somehow. Do you happen to have the parameters for your internet connection?

**Comment by Hank-Zhou** [comment]
waste time

**Comment by venkat-p-r** [comment]
Same problem here any solutions 

Failure Information:
	- http error: failed to reach URL https://cas.modular.com/0/LlIFrO9QZXgcwuWtdenG4czT5JznHEGIR5HEf3zp8jc with cURL error SSL connect error - Looking for LlIFrO9QZXgcwuWtdenG4czT5JznHEGIR5HEf3zp8jc

**Comment by liu20080331** [comment]
> Step-4: modular install mojo modular install mojo modular: error: invalid manifest: expiration has passed I require help here

Maybe try modular clear before running?

**Comment by bradleyayers** [comment]
I encountered this issue when I terminated `modular install mojo` after running it for a few seconds. From then on running `modular install mojo` would run for 5 seconds and then exit with

```
modular: error: invalid manifest: targets manifest was not an object
```

It was fixed by running `modular clean && modular install mojo`.

Perhaps it partially downloaded some files that became corrupted when I terminated the process.
